### ISSUE TITLE
Dataset images type

### ISSUE THREAD

**Issue by rasmusGrift** [wonbeomjang/mobile-hair-segmentation-pytorch#7]
Hey!

I'm currently trying to train with your project. The problem is that I don't get any predictions out of it. I'm not sure if I'm using the correct input images.

![epoch-4_step-0](https://user-images.githubusercontent.com/23166599/78019796-87bb5300-7350-11ea-8aef-e8f7953b5aee.png)

What kind of images does the dataloader expect?
I'm now using .png as normal image and the mask is also a png converted('L') where 0 is background and 255(white) is the object.
I mention I added converted('L') because I had to change this line in the loader to work

`mask = Image.open(os.path.join(self.data_folder, 'mask_right', self.image_name[index])).convert('L')`

**Comment by wonbeomjang** [comment]
This model expect that 3x224x224 RGB image for original image and 2x224x224 for mask image (0 for background and 255 for hair)
